src/pages/home/components/events/events_data.js

```diff
@@ -0,0 +1,47 @@
+// import RumahHantu from "../../../../assets/events/rumah_hantu.jpg";
+
+/**
+ * events field:
+ * - desc
+ * - img
+ * - link
+ */
+
+export const UPCOMINGEVENTS = {
```

Comment:
If you decided to use the map thing that I mentioned above, just do `export const EVENTS = [UPCOMINGEVENTS, RANTANGAN]`